bitnami/rabbitmq/templates/_helpers.tpl

```diff
@@ -240,7 +259,7 @@ otherwise it generates a random value.
     {{- $len := (default 16 .Length) | int -}}
     {{- $obj := (lookup "v1" "Secret" .Namespace .Name).data -}}
     {{- if $obj }}
-        {{- index $obj .Key | b64dec -}}
+        {{- index $obj .Key | trimAll "\"" | b64dec -}}
```

Comment:
this trimAll was on line 9 in secrets.yaml; I believe its to trim accidently added quotations in passwords.